src/ncp/ncp_uart.cpp

```diff
@@ -287,6 +287,8 @@ void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat
     }
     va_end(args);
 
+    otNcpStreamWrite(0, reinterpret_cast<const uint8_t*>("\r\n"), 2);
```

Comment:
@nibanks thanks for sharing the situation on windows. However with wpantund, log messages seems not outputted in time, instead, they are mixed and outputted with one `STREAM_DEBUG`, somethings messages are not complete. @abtink do you encounter the same situation I described? If not, am I missing any configurations?